FSQCellManifest/FSQCellRecord.h

```diff
@@ -88,20 +90,20 @@
 @property (nonatomic, copy) NSString *reuseIdentifier;
 
 /**
- This contents of this dictionary are not used internally by the manifest classes. 
+ This contents of this dictionary are not used internally by the manifest classes.
  You can use it to attach arbitrary data to the cell record for your own later use.
  */
-@property (nonatomic, readonly) NSMutableDictionary *userInfo;
+@property (nonatomic, readonly) NSMutableDictionary<NSString *, id> *userInfo;
 
 /**
  Convenience initializer with the most commonly set properties as method parameters
  
  You must include a cellClass. All other parameters are optional.
  */
-- (instancetype)initWithModel:(id)model 
-                    cellClass:(Class)cellClass 
-                  onConfigure:(FSQCellRecordConfigBlock)onConfigure
-                  onSelection:(FSQCellRecordSelectBlock)onSelection;
+- (instancetype)initWithModel:(id)model
```

Comment:
Oof :( so that will cause problems with quite a bit existing code where we do have static always-appears-a-certain-way cells that use a nil model